examples/CH4-refact-dpdispatcher/machine-ali-ehpc.json

```diff
@@ -46,7 +46,11 @@
         "queue_name": "T4_4_15",
         "group_size": 5,
         "source_list": ["/home/fengbo/deepmd.1.2.4.env"]
-      }
+      },
+      "_comments" : "In user_forward_files, define input files to be uploaded.",
+      "user_forward_files" : [],
+      "_comments" : "In user_backward_files, define output files to be collected.",
+      "user_backward_files" : ["HILLS"]
```

Comment:
Shouldn't it be model_devi_user_backward_files?

Reply:
No. Should be user_backward_files. In machine.json, we have left only three keys, `train`, `model_devi` and `fp`, each is a list of dict.  Each dict is a complete machine parameter, the design of list serves to keep the possibility for multiple machines.  Please refer to https://github.com/deepmodeling/dpgen/blob/devel/examples/CH4-refact-dpdispatcher/machine-dpcloudserver.json .  Old examples have been deprecated for a long time.

While in the main codes dpdispatcher still read the keys "model_devi_resources", "model_devi_machines",  we do a conversion in `convert_mdata` (in the past we use decide_fp_machine, decide_train_machine and decide_model_devi_machine ). In refactoring codes, the variables such as "model_devi_resources" should also be deprecated. Otherwise it's difficult to modularize the codes and generalize.